### ISSUE TITLE
Error: The `Style/BracesAroundHashParameters`

### ISSUE THREAD

**Issue by skoomer** [rubocop/rubocop#9651]
vendor/bundle/ruby/3.0.0/gems/jwt-2.2.2/.rubocop.yml: Style/CaseIndentation has the wrong namespace - should be Layout
vendor/bundle/ruby/3.0.0/gems/jwt-2.2.2/.rubocop.yml: Style/SpaceInsideHashLiteralBraces has the wrong namespace - should be Layout
vendor/bundle/ruby/3.0.0/gems/jwt-2.2.2/.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout
vendor/bundle/ruby/3.0.0/gems/jwt-2.2.2/.rubocop.yml: Lint/EndAlignment has the wrong namespace - should be Layout
vendor/bundle/ruby/3.0.0/gems/jwt-2.2.2/.rubocop.yml: Style/MultilineMethodCallIndentation has the wrong namespace - should be Layout
vendor/bundle/ruby/3.0.0/gems/jwt-2.2.2/.rubocop.yml: Style/MultilineOperationIndentation has the wrong namespace - should be Layout
Error: The `Style/BracesAroundHashParameters` cop has been removed. Please use  and/or  instead.
(obsolete configuration found in vendor/bundle/ruby/3.0.0/gems/jwt-2.2.2/.rubocop.yml, please update it)

--------

After connecting to Circleci
I spent a lot of time but I did not find how to solve this problem

## Actual behavior
 `rubocop --debug`
26 files inspected, no offenses detected
Finished in 11.325489999959245 seconds

## RuboCop version

```

1.12.0 (using Parser 3.0.0.0, rubocop-ast 1.4.1, running on ruby 3.0.0 x86_64-darwin18)
  - rubocop-rails 2.9.1
  - rubocop-rspec 2.2.0
```

```
require:
  - rubocop-rspec
  - rubocop-rails
  - rubocop-performance

AllCops:
  NewCops: enable
  Exclude:
    - 'config/**/*'
    - 'db/**/*'
    - 'script/**/*'
    - 'bin/**/*'
    - vendor/bundle/**/*
    - 'bin/{rails,rake}'
    - 'node_modules/**/*'
    - 'spec/rails_helper.rb'
    - 'Gemfile'
    - '!ruby/regexp /old_and_unused\.rb$/'
    

Metrics/BlockLength:
  Exclude:
    - config/**/*
    - 'spec/**/*'
    - 'vendor/*'
    - 'vendor/**/*'
    - 'node_modules/**/*'

Style/Documentation:
  Enabled: false

Style/Encoding:
  Enabled: false

Style/FrozenStringLiteralComment:
  Enabled: false
```

**Comment by dvandersluis** [comment]
Can you share your `.rubocop.yml`? It looks like it's running rubocop on vendored gems (with their configs, which are not up-to-date). You should probably add

```yaml
AllCops:
  Exclude:
  - vendor/bundle/**/*
```

to your config (or merge it into what you have in there).

**Comment by skoomer** [comment]
> Can you share your `.rubocop.yml`? It looks like it's running rubocop on vendored gems (with their configs, which are not up-to-date). You should probably add
> 
> ```yaml
> AllCops:
>   Exclude:
>   - vendor/bundle/**/*
> ```
> 
> to your config (or merge it into what you have in there).

Thank you, updated)

**Comment by tejasbubane** [comment]
> Thank you, updated)

@skoomer Are you still seeing the error after excluding `vendor/bundle`?

**Comment by skoomer** [comment]
> > Thank you, updated)
> 
> @skoomer Are you still seeing the error after excluding `vendor/bundle`?

It's work for me, thank you
